tests/python/unittest/test_tf_model.py

```diff
@@ -0,0 +1,60 @@
+# coding: utf-8
+from dlr import TFModelImpl
+import numpy as np
+
+FROZEN_GRAPH_PATH = "/tmp/test_graph.pb"
+
+
+def generate_frozen_graph():
+    import tensorflow as tf
+    a = tf.placeholder(tf.float32, shape=[2, 2], name="input1")
+    b = tf.placeholder(tf.float32, shape=[2, 2], name="input2")
+    mm = tf.matmul(a, b)
+    tf.square(mm, name="output1")
+    mm_flat = tf.reshape(mm, shape=[-1])
+    tf.argmax(mm_flat, name="output2")
+
+    with tf.Session() as sess:
+        output_graph_def = tf.compat.v1.graph_util.convert_variables_to_constants(
+            sess,
+            tf.get_default_graph().as_graph_def(),
+            ["output1", "output2"]
+        )
+
+        with tf.gfile.GFile(FROZEN_GRAPH_PATH, "wb") as f:
+            f.write(output_graph_def.SerializeToString())
+        print("%d ops in the final graph." % len(output_graph_def.node))
+
+
+def test_tf_model(dev_type=None, dev_id=None):
```

Comment:
it would be good to add more test cases, to cover the pb file of a real model like mobilenet.

Reply:
It will be done in the integration test. Out current approach with real model tests is to store model files on S3. Integration test downloads model file and runs the test.
This particular test is unit test. Unit tests should be small, fast and do not need internet connection.